modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectActionLocalServiceImpl.java

```diff
@@ -383,6 +418,16 @@ private void _validateExternalReferenceCode(
 		}
 	}
 
+	private void _validateInvokerBundle(String messageType, boolean system)
```

Comment:
Refact this method like [this one](https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java#L594-L602)